src/main/java/com/teragrep/rlp_03/frame/delegate/event/RelpEventOpen.java

```diff
@@ -45,29 +45,51 @@
  */
 package com.teragrep.rlp_03.frame.delegate.event;
 
-import com.teragrep.rlp_01.RelpCommand;
-import com.teragrep.rlp_01.RelpFrameTX;
+import com.teragrep.rlp_03.frame.RelpFrame;
+import com.teragrep.rlp_03.frame.RelpFrameImpl;
 import com.teragrep.rlp_03.frame.delegate.FrameContext;
+import com.teragrep.rlp_03.frame.fragment.Fragment;
+import com.teragrep.rlp_03.frame.fragment.FragmentFactory;
+import com.teragrep.rlp_03.frame.fragment.FragmentStub;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
-import java.util.ArrayList;
-import java.util.List;
+public final class RelpEventOpen extends RelpEvent {
 
-public class RelpEventOpen extends RelpEvent {
+    private static final Logger LOGGER = LoggerFactory.getLogger(RelpEventOpen.class);
+    private final FragmentFactory fragmentFactory;
+    private final RelpFrame responseFrameTemplate;
 
-    private static final String responseData = "200 OK\nrelp_version=0\n"
-            + "relp_software=RLP-01,1.0.1,https://teragrep.com\n" + "commands=" + RelpCommand.SYSLOG + "\n";
+    public RelpEventOpen() {
+        this.fragmentFactory = new FragmentFactory();
+        // TODO create FragmentFactory
+        Fragment txn = new FragmentStub();
+        Fragment command = fragmentFactory.create("rsp");
+        String content = "200 OK\nrelp_version=0\nrelp_software=RLP-01,1.0.1,https://teragrep.com\ncommands=syslog\n";
```

Comment:
This version string doesn't seem to be correct anymore

Reply:
fixed in #208